lib/ledger.js

```diff
@@ -14,6 +14,7 @@ const {promisify} = require('util');
 const getAgentIterator = promisify(brLedgerAgent.getAgentIterator);
 const {config, util: {delay, BedrockError}} = bedrock;
 const {WebLedgerClient} = require('web-ledger-client');
+const bedrockHttpsAgent = require('bedrock-https-agent');
```

Comment:
Most bedrock code uses the shorter `brHttpsAgent` style.

Reply:
Addressed here: https://github.com/digitalbazaar/bedrock-ledger-core/pull/5/commits/79ac0d07537c4309a98edc607f52ad46d74cb70d